### PR TITLE
systematic period usage

### PR DESCRIPTION
**Pull request by raz0rknaif**
if sentence is complete, a period is used. Otherwise, periods are not used. Note an incomplete sentence will use a period if there are multiple sentences in the string.

i was quite confused about how to use periods. this is done to hopefully allay confusion in regards to using them.

for example, in OSaveGameProvider, there was previously:

Path too long to the saved game.
Path too long to the saved game

### CODE REVIEW

**Comment by raz0rknaif** [comment]
"Incorrect kingdom information received from the host." This is a complete sentence. I shouldn't have removed the period if the stated rules apply.

File: src/OTALKRES.cpp [hunk 1]
```diff
@@ -1087,7 +1087,7 @@ int TalkRes::detect_talk()
 		choice_question_second_line = NULL;
 
 		talk_choice_count = 0;
-		add_talk_choice( _("Continue"), 0 );
+		add_talk_choice( _("Continue."), 0 );
```
Comment (the3dfxdude) [inline src/OTALKRES.cpp]: What rule does this follow? This is more of a "button" than a sentence.
Yes I know that this change is consistent with the rest of the talk choices.

**Comment by raz0rknaif** [comment]
Yes, I know. while the Russian translators probably translated this as an
infinitive, I consider this verb an imperative. I. e. a complete sentence.
I can't say I'm an expert in English imperatives tho. Let me check...

**Comment by the3dfxdude** [comment]
I merged the changes with the one correction you mentioned.

Couple thoughts:

You don't need to worry about the ERR() and MSG() messages at all. They are not translated, and do not show up anywhere a normal user would see (not normally built into the game, and sent to somewhere outside the game). Same with Log and LongLog etc.

Some of these are error/debug dialogs. They don't normally happen either, but they are built into the game so they do need translation just in case. These aren't super important visually.

**Pull request closed without merging**